lib/prompt-library-params.js

```diff
@@ -69,8 +71,23 @@ module.exports = async (opts) => {
         type: 'list',
         name: 'template',
         message: 'Template',
-        choices: [ 'default', 'typescript' ],
+        choices: [ 'default', 'typescript', 'custom' ],
         default: opts.template
+      },
+      {
+        type: 'input',
+        name: 'templatePath',
+        message: 'Template Path',
+        when: ({ template }) => template === 'custom',
+        validate: input => new Promise(resolve => {
+          const fullPath = path.resolve(process.cwd(), input)
+          fs.stat(fullPath, (err, stats) => {
+            if (err) {
+              resolve(`Cannot resolve directory at: ${fullPath}`)
```

Comment:
this should `return resolve(...)` here, otherwise it will resolve the promise twice. this second resolve will technically be ignored but is considered bad practice.

Reply:
Ahh, good catch! Fixed